README.md

```diff
@@ -56,6 +56,57 @@ issue](https://github.com/esPass/pretix-eth-payment-plugin/pull/49)
 6. Head to the plugin settings page to set the deposit address for both
    Ethereum and DAI.
 
+## Automatic payment confirmation with the `confirm_payments` command
+
+This plugin includes a [django management
+command](https://docs.djangoproject.com/en/2.2/howto/custom-management-commands/#module-django.core.management)
+that can be used to automatically confirm orders from Ethereum transactions and
+ERC20 token transfers.  Here's are some example invocations of this command:
+```bash
+# Using the pretix module
+python -m pretix confirm_payments --event-slug=devcon-5
+
+# Using a django manage.py file
+python manage.py confirm_payments --event-slug=devcon-5
+```
+Above, the `confirm_payments` command uses the `--event-slug` argument to
+determine the wallet address to which ticket payments for the `devcon-5` event
+were sent.  It then inspects *all* external and internal transactions sent to
+the event's wallet address to determine if sufficient payments were made for
+payment records identified by the payment IDs in encoded the transactions' wei
+values.  It also inspects all token transfer events targeting the event's
+wallet address for the DAI stablecoin's mainnet contract address.
+Alternatively, the same command above could have been invoked as follows:
+```bash
+python manage.py confirm_payments --wallet-address=<devcon-5-wallet-address>
+```
+...where `<devcon-5-wallet-address>` is replaced with the explicit
+`0x`-prefixed wallet address for the Devcon 5 event.
+
+The `confirm_payments` command also supports a number of other arguments.  Here
+are some example uses of them:
+```bash
+python manage.py confirm_payments \
+    --event-slug=<slug> \
+    --token-address=<token-address> \
+    --api=blockscout-goerli \
+    --start-block=<start-block> \
+    --end-block=<end-block> \
+    --dry-run
```

Comment:
Do you think it makes sense to invert this and have the default behavior to perform a dry run followed by a confirmation prompt that would then confirm everything that was run through the dry run?  That would probably imply a `--no-dry-run` flag to allow bypassing the prompt and the `--dry-run` flag simply running it as a dry run but exiting without prompting.

Reply:
Yeah, I like that idea. However, I think we get effectively the same functionality by just adding the `--no-dry-run` flag instead of adding a prompt. The UX is *slightly* worse but the code doesn't get any more complex.

Reply:
Here's more of my thinking about not needing a prompt:

1. It seems like the only difference between having a prompt and defaulting to performing dry runs is that you would need to re-run the command with `--no-dry-run` instead of just pressing `y<enter>`.  One is *slightly* less convenient than the other but not by much.  I it's possible that someone might do a dry run without realizing it, but the risk there seems low since the documentation is pretty clear and they'd probably realize what's happening pretty quickly.
2. I imagine it's more likely that the command would be run by a cron-job in which case you wouldn't need or want a prompt.